### ISSUE TITLE
3.11 emits a warning during build

### ISSUE THREAD

**Issue by ef4** [emberjs/data#6218]
Reproduction:

1. Using ember-cli 3.10.1 (which is the latest stable as of this writing), run `ember new example-app`.
2. `cd example-app`
3. `ember install ember-data` (I got 3.11.0).
4. `ember build`

You will see this warning printed in the terminal:

```
'ember-inflector' is imported by -private/build-url-mixin.js, but could not be resolved – treating it as an external dependency
```

which I think comes from Ember Data's rollup.

Probably `ember-inflector` *is* supposed to be an external dependency, but it needs to be configured explicitly so rollup doesn't warn.

**Comment by efx** [comment]
I see this on master as well as of 2c2cfb99ba16232fbc779371acc2e14415cfe484.

**Comment by runspired** [comment]
hrm, I swear I zapped all of these prior to releasing 3.11. Will take a poke at it later.

**Comment by jameshahn2** [comment]
Getting the same warning after upgrading to 3.11.

**Comment by rwjblue** [comment]
Will be fixed by #6242
